### PR TITLE
Add damaging and goodfaith models for huwiki 

### PR DESCRIPTION
**Pull request by tgr**
Also update the list of trusted groups.

Bug: [T185903](https://phabricator.wikimedia.org/T185903)

### CODE REVIEW

File: config/wikis/huwiki.yaml [hunk 1]
```diff
@@ -15,6 +15,9 @@ trusted_groups:
     - checkuser
     - abusefilter
     - bureaucrat
+    - editor
+    - templateeditor
+    - interface-editor
```
Comment (halfak) [inline config/wikis/huwiki.yaml:20]: Adding these now will cause some warnings, but no big issues.  Essentially, we'll have labels for some edits where "needs_review" is false with "reason" == "trusted_group"

File: Makefile [hunk 2]
```diff
@@ -1459,11 +1459,79 @@ models/huwiki.reverted.rf.model: \
...
+tuning_reports/huwiki.damaging.md: \
+		datasets/huwiki.labeled_revisions.w_cache.40k_2016.json
+	cat $< | \
+	revscoring tune \
+		config/classifiers.params.yaml \
+		editquality.feature_lists.huwiki.damaging \
+		damaging \
+		roc_auc.labels.true \
+		--label-weight "true=$(damaging_weight)" \
+		--pop-rate "true=0.0872" \
```
Comment (halfak) [inline Makefile]: 1/10th of all edits are damaging?  that seems wrong.

**Comment by tgr** [comment]
@halfak pointed out that I used the percentage of hand-labeled edits where I should have used the percentage of total edits so I redid the commits using the ratio between

     ack '"goodfaith": true' datasets/huwiki.labeled_revisions.40k_2016.json | wc -l

and

    ack '"rev_id"' datasets/huwiki.labeled_revisions.40k_2016.json | wc -l

I'll update the docs once the PR passes review.

**Comment by tgr** [comment]
`ack '"rev_id"' datasets/huwiki.labeled_revisions.40k_2016.json | wc -l` gives 39678 so I am not sure if that's the correct denominator or I should have used 40000 instead (although it's not much difference). I suppose the missing 322 revisions are deleted ones?

**Comment by halfak** [comment]
I'd recommend using the 39678 as the denominator -- but it shouldn't make too much of a difference.  I think you're right and 322 are revisions to deleted pages or they have been rev-deleted.

**Comment by adamwight** [comment]
Needs a rebase.  The master branch has an updated merge_labels algorithm which simplifies the Makefile logic.

File: config/wikis/huwiki.yaml [hunk 3]
```diff
@@ -30,3 +33,13 @@ models:
             min_samples_leaf: 13
         pop_rate_true: 0.014812583163867339
         build_number: 1
+    damaging:
```
Comment (adamwight) [inline config/wikis/huwiki.yaml]: This patch should drop the huwiki.reverted model too, AIUI.

**Comment by adamwight** [comment]
This needs a rebase onto improvements to merge_labels, which allows us to simplify the case.

**Comment by adamwight** [comment]
P.S., you should have privileges on this repo, and if the branch were under the org repo I could push my rebase: https://github.com/wiki-ai/editquality/compare/awight-huwiki

**Comment by tgr** [comment]
Closing in favor of #152 which uses a local branch (it did not seem possible to change the source for this one).

**Pull request closed without merging**